content/courses/advanced-ada/chapters/aggregates.rst

```diff
@@ -435,14 +435,339 @@ consider using interfaces instead of null records. We'll discuss this topic
 Extension Aggregates
 --------------------
 
-.. admonition:: Relevant topics
+Extension aggregates allow us to express information for record types with
+type extension using a single expression. They are similar to record
+aggregates, but include the :ada:`with` keyword, and are used to write more
+complex aggregates.
```

Comment:
Maybe something like this:
Extension aggregates provide a convenient way to express an aggregate for a type that extends -- adds components to -- some existing type (the "ancestor"). Although mainly a mater of convenience, an extension aggregate is essential when a client wants to express an aggregate for an extension of a private ancestor type, that is, when the client doesn't have compile-time visibility to the ancestor type's components.

Reply:
I like it. I'm using this now (with minor editorial changes).